src/test/ui/runtime/backtrace-debuginfo.rs

```diff
@@ -12,6 +12,7 @@
 // ignore-pretty issue #37195
 // ignore-emscripten spawning processes is not supported
 // ignore-sgx no processes
+// ignore-fuchsia Backtrace not symbolized, trace different line alignment
```

Comment:
What does "trace different line alignment" mean? As in this test checks the formatting of the backtrace and Fuchsia doesn't match?

Reply:
Correct -- and test failure comes from the lack of symbolization